pkg/openscap/openscap.go

```diff
@@ -14,121 +14,145 @@ import (
 	"syscall"
 	"time"
 
-	docker "github.com/fsouza/go-dockerclient"
+	"github.com/fsouza/go-dockerclient"
 	iiapi "github.com/openshift/image-inspector/pkg/api"
-	"github.com/openshift/image-inspector/pkg/util"
-	xmldom "github.com/subchen/go-xmldom"
+	"github.com/subchen/go-xmldom"
 )
 
 const (
-	CPE             = "oval:org.open-scap.cpe.rhel:def:"
-	CPEDict         = "/usr/share/openscap/cpe/openscap-cpe-oval.xml"
-	CVEUrl          = "https://www.redhat.com/security/data/metrics/ds/"
-	DistCVENameFmt  = "com.redhat.rhsa-RHEL%d.ds.xml.bz2"
-	ArfResultFile   = "results-arf.xml"
-	HTMLResultFile  = "results.html"
-	TmpDir          = "/tmp"
-	Linux           = "Linux"
-	OpenSCAP        = "openscap"
-	ImageShortIDLen = 11
-	Unknown         = "Unknown"
-	LinuxVersionPH  = "Unknown"
-	OpenSCAPVersion = "1.2"
-
-	CVEDetailsUrl = "https://cve.mitre.org/cgi-bin/cvename.cgi?name="
+	CVEUrl = "https://www.redhat.com/security/data/metrics/ds/"
+
+	cpe             = "oval:org.open-scap.cpe.rhel:def:"
+	cpeDict         = "/usr/share/openscap/cpe/openscap-cpe-oval.xml"
+	distCVENameFmt  = "com.redhat.rhsa-RHEL%d.ds.xml.bz2"
+	arfResultFile   = "results-arf.xml"
+	htmlResultFile  = "results.html"
+	linux           = "Linux"
+	scannerName     = "openscap"
+	imageShortIDLen = 11
+	unknown         = "Unknown"
+	linuxVersionPH  = "Unknown"
+	openSCAPVersion = "1.2"
+	cveDetailsUrl   = "https://cve.mitre.org/cgi-bin/cvename.cgi?name="
 )
 
 var (
-	RHELDistNumbers = [...]int{5, 6, 7}
-	osSetEnv        = os.Setenv
+	rhelDistNumbers = []int{5, 6, 7}
 )
 
-// rhelDistFunc provides an injectable way to get the rhel dist for testing.
-type rhelDistFunc func() (int, error)
-
-// inputCVEFunc provides an injectable way to get the cve file for testing.
-type inputCVEFunc func(int) (string, error)
-
-// chrootOscapFunc provides an injectable way to chroot and execute oscap for testing.
-type chrootOscapFunc func(...string) ([]byte, error)
-
-// setEnvFunc provides an injectable way to get the cve file for testing.
-type setEnvFunc func() error
```

Comment:
Ok, I see later on the tests were removed.  I remember that they were added there because we specifically ran into issues with it.  Maybe @simon3z remembers better but it may be useful to keep them around